dask_mpi/tests/test_cli.py

```diff
@@ -61,6 +61,26 @@ def test_no_scheduler(loop, mpirun):
                         sleep(0.2)
 
 
+@pytest.mark.parametrize("nanny", ["--nanny", "--no-nanny"])
+def test_non_default_ports(loop, nanny, mpirun):
+    with tmpfile(extension="json") as fn:
+
+        cmd = mpirun + ["-np", "2", "dask-mpi", "--scheduler-file", fn, nanny,
+                        "--scheduler-port", "56723",
+                        "--worker-port", "58464",
+                        "--nanny-port", "50164"]
+
+        with popen(cmd):
+            with Client(scheduler_file=fn) as c:
+
+                start = time()
+                while len(c.scheduler_info()["workers"]) != 3:
+                    assert time() < start + 10
+                    sleep(0.2)
+
+                assert c.submit(lambda x: x + 1, 10, workers="mpi-rank-1").result() == 11
```

Comment:
You should be able to drop the `workers="mpi-rank-1"` bit here.  I suspect that that was copied over from a previous test.

You might also want to investigate the contents of `c.scheduler_info()` so that you can test that the ports used above were in fact used.

Reply:
Thanks! I can confirm that the ports are indeed used - should I go ahead and add this to the test?

Reply:
Done.